src/OrbitGl/App.cpp

```diff
@@ -1459,8 +1459,25 @@ void OrbitApp::UpdateProcessAndModuleList(int32_t pid) {
         }
       }
       // (D) Load Modules again (and pass on functions to hook after loading)
-      LoadModules(modules_to_reload, std::move(function_hashes_to_hook_map),
-                  std::move(frame_track_function_hashes_map));
+      const auto future = LoadModules(modules_to_reload);
+
+      if (main_thread_executor_->WaitFor(future) != MainThreadExecutor::WaitResult::kCompleted) {
+        return;
+      }
+
+      for (const auto& [module_path, function_hashes] : function_hashes_to_hook_map) {
+        ModuleData* const module_data = GetMutableModuleByPath(module_path);
+        if (module_data == nullptr) continue;
+        (void)SelectFunctionsFromHashes(module_data, function_hashes);
```

Comment:
Shouldn't the returned error used (even if only for a log)?

Reply:
See me other comments. `SelectFunctionsFromHashes()` always returns success